BrowserKit/Sources/Shared/Strings.swift

```diff
@@ -5,11 +5,10 @@
 // swiftlint:disable line_length
 import Foundation
 
-// MARK: - Localization bundle setup
-class BundleClass {}
-
 public struct Strings {
-    public static let bundle = Bundle(for: BundleClass.self)
+    // This is a little workaround to get localizations, in the next PRs they will be moved in a spm package
+    // So we just call Bundle.module to reference the localizations
```

Comment:
@FilippoZazzeroni if this is a task, please add a ticket here even if you do plan on making another PR so we don't forget

Reply:
Sure @nbhasin2 i'll add the ticket there, it should be plan for part 2